pkg/pillar/types/global.go

```diff
@@ -332,7 +332,7 @@ var GlobalConfigMinimums = GlobalConfig{
 	DownloadRetryTime:       60,
 	DomainBootRetryTime:     10,
 	Dom0MinDiskUsagePercent: 20,
-	RktGCGracePeriod:        30,
+	RktGCGracePeriod:        43200,
```

Comment:
While 30 seconds might be too small as a minimum, having the min configrable value be 12 hours seems excessive. Wouldn't a min of 5 or 10 minutes make more sense?

Reply:
@eriknordmark Yes, we can set min value to 10 min. I will update it. Thanks.